controllers/grid/plugins/form/UploadPluginForm.inc.php

```diff
@@ -53,6 +53,8 @@ function readInputData() {
 	function fetch($request) {
 		$templateMgr = TemplateManager::getManager($request);
 		$templateMgr->assign('function', $this->_function);
+		$templateMgr->assign('category', $request->getUserVar('category'));
+		$templateMgr->assign('plugin', $request->getUserVar('plugin'));
```

Comment:
(No big deal, but I like using the `array` syntax for `$templateMgr->assign` when there are more than a couple of lines)